From 3a282f3e9b098fee8ee80c54f70e30cbc06fada4 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Tue, 1 Jan 2019 08:23:46 +0900
Subject: [PATCH] Makefile: update dist-clean

Update `make dist-clean` to remove `shared` directory in each of SPP
processes under `src`.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 GNUmakefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/GNUmakefile b/GNUmakefile
index 7c2b55a..4f04bba 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -25,6 +25,7 @@ DOC_ROOT = docs/guides
 dist-clean:
 	make clean
 	rm -rf $(wildcard src/*/$(RTE_TARGET))
+	rm -rf $(wildcard src/*/shared)
 
 .PHONY: doc
 doc: doc-all
-- 
2.7.4

